### PR TITLE
Added Gradle integration.

### PR DESCRIPTION
**Pull request by tromantic**
Mostly took the results from running:

``` shell
gradle init --type=pom
```

### CODE REVIEW

**Comment by cmosher01** [comment]
Please remove the pom.xml file.

**Comment by pcssi** [comment]
Hi @cmosher01, commiet: 85c05a6911391afddea13c8b61c8b0d697326eb2 removed the pom.xml file.

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+group = 'com.surveysampling.scm'
+version = '1.0.0-SNAPSHOT'
+
+description = 'Java program for Raspberry Pi (pi4j) to control a continuous integration traffic light via the GPIO pins.'
+
+sourceCompatibility = JavaVersion.VERSION_1_8
+targetCompatibility = JavaVersion.VERSION_1_8
+
+repositories {
+     maven { url "http://ctnexus01.surveysampling.com/content/groups/maven" }
```
Comment (tromantic) [inline build.gradle:13]: tl;dr `gradle init --type=pom`  uses the active profiles in `~/.m2/settings.xml` when generating `build.gradle`

I had an active profile for SSI that configured the Nexus repository in `~/.m2/settings.xml`. I was scratching my head after looking at commit: 537e84419999166db86f189649e044f0032110ab; wondering out how the Nexus address got in there.

Comment (cmosher01) [inline build.gradle:13]: Oh! OK, I see :)

**Pull request merged**